src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,313 @@
+---
+part_number: II
+chapter_number: 13
+title: Capabilities
+description: Capabilities chapter of the 2020 Web Almanac covering brand-new, powerful web platform APIs.
+authors: [christianliebel]
+reviewers: [logicalphase,tomayac]
+analysts: [tomayac]
+translators: []
+discuss: 2049
```

Comment:
FYI here's the URL for this discussion thread: https://discuss.httparchive.org/t/chapter-13-capabilities/2049

I'd recommend "watching" it so you can be notified of comments by readers.